vulyk/templates/base/_nav.html

```diff
@@ -0,0 +1,38 @@
+<li>
+    <a href="#tag_hints" class="popup-with-zoom-anim">
+        {% if config.LANGUAGE == 'ua' %}Інструкція
```

Comment:
I wanted to use Babel, but this is new dependency, and collaborators have to accept this new dependency.